components/map/Map.js

```diff
@@ -9,21 +10,22 @@ export default function Map({ users }) {
   ];
 
   return (
-    <MapContainer
-      center={[0, 0]}
-      zoom={2}
-      minZoom={2}
-      zoomControl={true}
-      scrollWheelZoom={true}
-      maxBounds={boundsMap}
-      maxBoundsViscosity={0.7}
-      style={{ height: "100vh" }}
-    >
-      <TileLayer
-        attribution='&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
-        url="https://b.tile.openstreetmap.org/{z}/{x}/{y}.png"
-      />
-      <Clusters users={users} />
-    </MapContainer>
+      <MapContainer
```

Comment:
Please fix the indenting to match the prior version so the change you made is easier to see.

Reply:
I think the intendation was same before also.

Reply:
According to the diff, all of the lines inside the MapContainer tag are indented 2 spaces more than before. This is probably due to the div wrapper that was removed.

Reply:
Ok, I got it

Reply:
Here is a diff showing the spacing changes that should be removed:
![MapContainer](https://github.com/EddieHubCommunity/LinkFree/assets/35927536/181e1764-751f-4579-b9a2-2115bb613ff8)